gradle.properties

```diff
@@ -1,18 +0,0 @@
-# Project-wide Gradle settings.
```

Comment:
I added this file to .gitignore, it contains our sonatype credentials and gnupg secret.